docs/product-installation-guide/src/main/asciidoc/clustering-elasticsearch-proc.adoc

```diff
@@ -15,7 +15,7 @@ transport.tcp.port: 9300
 xpack.security.enabled: false
 discovery.zen.minimum_master_nodes: 1
 ----
-. Install the following plugin:
+. Install the following plug-in:
 +
 * On Linux or UNIX-based systems, enter:
```

Comment:
Several instances of "enter:". I don't have a problem, but IBM would go with a complete sentence before the colon, something like "...enter the following command:". What you also do in this doc elsewhere is just "On Linux or UNIX-based systems:" and that's it. That works too. I'd just pick a style and stick to it.

Granted, you know that I'm guilty of flouting IBM style suggestions as an economy in certain cases. Up to you. Just pointing it out mainly for consistency.